paper/paper.md

```diff
@@ -49,27 +49,27 @@ On top of this, we added particles modelling the growth of sugar kelp which are
 Thanks to Julia's speed and efficiency the above model (1 km × 1 km × 100 m with 64 × 64 × 16 grid points) took about 30 minutes of computing time to simulate 10 days of evolution on an Nvidia P100 GPU. Figure made with `Makie.jl` [@makie]. \label{fig1}](eady_example.png)
 
 ``OceanBioME.jl`` is built with a highly modular design that allows user control and customization.
-There are three distinct module types implemented in ``OceanBioME.jl``:
+There are two distinct module types implemented in ``OceanBioME.jl``:
 
-- First, tracer-based ecosystem modules are formulated as a set of coupled ordinary differential equations.
+- First, tracer-based ecosystem modules are formulated in `AdvectedPopulations` as a set of coupled ordinary differential equations.
 These equations can be solved by ``OceanBioME.jl`` as box models, which is particularly useful for testing.
 The same modules can be integrated by ``Oceananigans.jl`` to provide tracer-based ecosystem models.
 
-- Second, boundary modules contain sets of equations which provide information at the top and bottom of the ocean.
-For example, air-sea gas exchange modules calculate the flux of carbon dioxide and oxygen at the sea surface, while sediment modules calculate fluxes of carbon and oxygen at the seafloor.
-
-- The third module type is "biologically active" particles.
+- The second module type is `Individual` "biologically active" particles.
 These consist of individual-based models which are solved along particle paths and can be coupled with the tracer-based modules and physics from ``Oceananigans.jl``.
 The biologically active particles can be advected by the currents, and/or they can move according to prescribed dynamics.
 For example, migrating zooplankton or fish can be modelled with biologically active particles and ``OceanBioME.jl`` allows these to interact with tracer-based components such as phytoplankton or detritus.
 
+`AdvectedPopulations` are supported by `Boundaries` modules which provide information at the top and bottom of the ocean. For example, the GasExchange submodule calculate the flux of carbon dioxide and oxygen at the sea surface, while sediment modules calculate fluxes of carbon and oxygen at the seafloor.
+
 We provide a simple framework and utilities (such as light attenuation integration) to build the necessary components of biogeochemical models.
-With the provided models, currently a simple Nutrient-Phytoplankton-Zooplankton-Detritus [@npzd] model, an intermediate complexity model, LOBSTER [@lobster], and a higher complexity model, PISCES [@pisces]. 
+With the provided models, currently a simple Nutrient-Phytoplankton-Zooplankton-Detritus [@npzd] model, and an intermediate complexity model, LOBSTER [@lobster]. 
```

Comment:
This sentence reads weird... Seems like you meant to write "with the provided models ... we have ..."?

Reply:
Oh yeah, definitely